examples/nimble-typescript/README.md

```diff
@@ -0,0 +1,117 @@
+# Nimble Typescript Getting Started
+
+This example project creates a simple counter application using Vanilla (no framework) Typescript and the Nimble Design System.
+
+## Prerequisites
+
+1. Install VSCode  (or your editor of choice) - <https://code.visualstudio.com/>  
```

Comment:
I'd vote to use markdown links rather than showing entire URLs, but could be convinced otherwise if you have rationale for this approach.
```suggestion
1. [Install VSCode](https://code.visualstudio.com/)  (or your editor of choice) 
```